components/scream/src/share/grid/point_grid.cpp

```diff
@@ -66,7 +66,7 @@ PointGrid::get_3d_vector_layout (const bool midpoints, const FieldTag vector_tag
 bool PointGrid::
 valid_geo_data (const std::string& name, const geo_view_type& /* data */) const {
   // Sanity checks
-  return name=="lat" || name=="lon" || name=="area";
+  return name=="lat" || name=="lon" || name=="area" || name=="hyam" || name=="hybm";
```

Comment:
Same here. I'm leaning on removing this "validation" routines at some point...

Reply:
Agree again. The more fields that are added, the messier and more unreadable this would become.